### PR TITLE
[DCOS-44856] Release Notes for Spark 2.5.0-2.2.1-3

### PR DESCRIPTION
**Pull request by soumasish**
## Description
Release notes for Spark 2.5.0-2.2.1-3
https://jira.mesosphere.com/browse/DCOS-44856

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/spark/2.5.0-2.2.1/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+title: Spark 2.5.0-2.2.1
+navigationTitle: Spark 2.5.0-2.2.1
+menuWeight: 1
+excerpt: Documentation for DC/OS Apache Spark 2.5.0-2.2.1
+model: /services/spark/data.yml
+render: mustache
+featureMaturity:
+---
+
+Welcome to the documentation for the DC/OS {{ model.techName }}. For more information about new and changed features, see the [release notes](/services/spark/2.4.0-2.2.1-3/release-notes/).
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/index.md]: This should be `2.5.0` release notes url

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+excerpt: Release notes for DC/OS Apache Spark 2.5.0-2.2.1
+featureMaturity:
+---
+
+## Version Spark and Spark History 2.5.0-2.2.1
+
+### New Features
+- Unique Mesos Task IDs for Spark executors
+- Trusted Ubuntu 18.04 base Docker image
+- `nobody` user support on RHEL/CentOS (via configuration)
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Compare previous release notes to this version. These should be worded similarly eg 
- `Added unique Mesos Task IDs for Spark executors`
- `Added trusted Ubuntu 18.04 base Docker image`... etc

Same for updates:
- `Updated JRE to 1.8.192`
- `Upgraded Ubuntu to 18.04`

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+- ZooKeeper dependencies update from 3.4.6 to 3.4.13
+
+
+### Bug fixes
+- `dcos task log` now works because of unique Mesos Tasks IDs of Spark executors
+- Fixed unstable health checks for Spark Dispatcher and History Server
+- Spark dispatcher task output now redirected to stdout and available in logs
+
+### Breaking Changes
+- new configuration option added: ‘docker_user’ which allows to override a user when running Spark using Docker containerizer
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Capitalize and clean up a little. Suggestion:

```
We've added a new configuration option: `docker_user` which enables overriding the default Spark user inside Docker containerized jobs
```

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+- Spark default Hadoop dependency is updated from 2.6 to 2.7
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Suggestion: `The default Hadoop dependency is now 2.7, not 2.6`

File: pages/services/spark/2.5.0-2.2.1/index.md (hunk 1 above)
Comment (soumasish) [reply pages/services/spark/2.5.0-2.2.1/index.md]: Done

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md (hunk 2 above)
Comment (soumasish) [reply pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Done

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md (hunk 3 above)
Comment (soumasish) [reply pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Done

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md (hunk 4 above)
Comment (soumasish) [reply pages/services/spark/2.5.0-2.2.1/release-notes/index.md]: Done

File: pages/services/spark/2.4.0-2.2.1-3/release-notes/index.md [hunk 5]
```diff
@@ -10,20 +10,20 @@ featureMaturity:
 ## Version Spark and Spark History 2.4.0-2.2.1-3
 
 ### New Features
-- Added service name to dispatcher’s VIP endpoints 
-- Added shell-escape fix to spark-cli (SPARK-21014) 
-- Added spark.mesos.executor.gpus (SPARK-21033) 
-- Added dispatcher and driver metrics 
-- Added statsd sink for spark metrics 
+- Added service name to dispatcher’s VIP endpoints
```
Comment (samvantran) [inline pages/services/spark/2.4.0-2.2.1-3/release-notes/index.md]: Not this file but there are two empty files that should be deleted 
`pages/services/spark/2.4.0-2.2.1-3/2.5.0-2.2.1`
`pages/services/spark/2.5.0-2.2.1/2.5.0-2.2.1`

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+### New Features
+- Added unique Mesos Task IDs for Spark executors
+- Added trusted Ubuntu 18.04 base Docker image
+- Added `nobody` user support on RHEL/CentOS (via configuration)
+
+### Updates
+- Changed default user for Docker container from ‘root’ to ‘nobody’
+- Updated JRE upgrade to 1.8.192
+- Updated Ubuntu upgrade 18.04
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]:
```suggestion
- Upgraded to Ubuntu 18.04
```

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+## Version Spark and Spark History 2.5.0-2.2.1
+
+### New Features
+- Added unique Mesos Task IDs for Spark executors
+- Added trusted Ubuntu 18.04 base Docker image
+- Added `nobody` user support on RHEL/CentOS (via configuration)
+
+### Updates
+- Changed default user for Docker container from ‘root’ to ‘nobody’
+- Updated JRE upgrade to 1.8.192
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]:
```suggestion
- Updated JRE to `1.8.192`
```

File: pages/services/spark/2.5.0-2.2.1/release-notes/index.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+- Updated ZooKeeper dependencies from 3.4.6 to 3.4.13
+
+
+### Bug fixes
+- `dcos task log` now works because of unique Mesos Tasks IDs of Spark executors
+- Fixed unstable health checks for Spark Dispatcher and History Server
+- Spark dispatcher task output now redirected to stdout and available in logs
+
+### Breaking Changes
+- We've added a new configuration option added: ‘docker_user’ which allows to override a user when running Spark using Docker containerizer
```
Comment (samvantran) [inline pages/services/spark/2.5.0-2.2.1/release-notes/index.md]:
```suggestion
- We've added a new configuration option: ‘docker_user’ which allows overriding the user when running Spark using Docker containerizer
```

**Comment by soumasish** [comment]
Thanks @samvantran

**Comment by lsgunnlsgunn** [comment]
Closing this PR due to merge conflicts.  I cherry-picked the commits from this to a working local branch. Please see this PR for the update:
https://github.com/mesosphere/dcos-docs-site/pull/1780

**Pull request closed without merging**